src/Client/components/DateTimeFormat.tsx

```
@@ -67,8 +67,15 @@ export function calculateDuration(start: string, end: string): string {
 
   return `${hours}h ${minutes}m ${seconds}s`;
 }
-export function parseTime(time: string): Time | null {
-    if (!time) return null;
-    const [hour, minute] = time.split(":").map(Number);
-    return new Time(hour, minute);
+export function parseTime(timeString: string): Time | null {
+    if (!timeString) return null;
+    const [timePart, period] = timeString.split(' ');
+    const [hours, minutes] = timePart.split(':');
+    let hour = parseInt(hours, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)